frontend/src/utils/aboutData.ts

```diff
@@ -4,6 +4,104 @@ export const aboutText = [
   'The code is licensed under the MIT License, encouraging contributions while protecting the authors from legal claims. All OWASP Nest leaders are certified ISC2 professionals and OWASP members who adhere to the OWASP Code of Conduct.',
 ]
 
+// TypeScript interfaces
+export interface KeyFeature {
+  title: string
+  description: string
+}
+
+export interface ProjectHistory {
+  year: string
```

Comment:
Having just year is not enough. Let's add at least a month name for better timeline representation.